### ISSUE TITLE
[Bug]: Fails to build when used with yarn 3 pnp mode

### ISSUE THREAD

**Issue by Talaxy009** [walinejs/waline#1832]
### 问题描述 | Describe the bug

I am trying to use Waline with my website (using yarn 3 with pnp mode) and it will fail to build. Only after I set nodeLinker to `node-modules` does it build successfully.

Here is the reproduction url: <https://codesandbox.io/p/sandbox/brave-cartwright-c2j1vt>

And you will see something like see: 
![image](https://user-images.githubusercontent.com/41264828/221894214-de1ffb7f-15e2-4efe-88bd-cbce0b7d017e.png)

### 问题网站 | Website URL

https://codesandbox.io/p/sandbox/brave-cartwright-c2j1vt

### 服务部署在哪里？ | Where your waline deploy?

Vercel (Default)

### 数据存储在哪里？| Where your comment data store?

LeanCloud(https://leancloud.app)

**Comment by Mister-Hope** [comment]
The repo itself is using pnpm. which has the same logic with yarn berry.

The problem in your log seems that vue-demi fails to access vue, so if you inisit there are issues, open an new issue to `vue-demi`.

**Comment by Talaxy009** [comment]
Thanks, I will issue it to vue-demi.

**Comment by Talaxy009** [comment]
My issue was resolved by adding these to my `.yarnrc.yml` file.

```yml
packageExtensions:
  'vue-demi@*':
    dependencies:
      'vue': '*'
```
